pvlib/irradiance.py

```diff
@@ -1621,7 +1621,7 @@ def disc(ghi, solar_zenith, datetime_or_doy, pressure=101325,
         pd.DatetimeIndex.dayofyear, or pd.DatetimeIndex.
 
     pressure : None or numeric, default 101325
-        Site pressure in Pascal. If None, relative airmass is used
+        Site pressure in Pascal. If not specified, relative airmass is used
```

Comment:
```suggestion
        Site pressure in Pascal. If specified, relative airmass is used 
```

Reply:
I think you misread this statements. Mind double-checking it?

Reply:
The logic in the code is perhaps not as clear as it could be. Relative air mass is used when ``pressure`` is None. Let's  clear this up in a new PR.

```suggestion
        Site pressure in Pascal. If not specified, relative airmass is used.
```